packages/ag-grid-theme/css/salt-ag-theme.scss

```diff
@@ -579,7 +579,7 @@ $ag-theme-salt-params-touch-dark: map-merge(
   );
   --agGrid-checkbox-foreground-selected: var(
     --saltAgGrid-checkbox-foreground-selected,
-    var(--salt-selectable-primary-foreground-selected)
+    var(--salt-actionable-primary-foreground-active)
```

Comment:
is this a different color? if so will need changeset for ag grid theme

Reply:
they're the same, both reference the same palette token `--salt-palette-interact-primary-foreground-active`